hazelcast/src/test/java/com/hazelcast/query/impl/QueryEntryTest.java

```diff
@@ -48,7 +53,7 @@ public void getAttribute_whenValueIsPortableObject_thenConvertedToData() {
         Data key = serializationService.toData("indexedKey");
 
         Portable value = new SampleObjects.PortableEmployee(30, "peter");
-        QueryEntry queryEntry = new QueryEntry(serializationService, key, value, Extractors.empty());
+        QueryableEntry queryEntry = entry(serializationService, key, value, Extractors.empty());
```

Comment:
why does serialization service need to be passed as argument since it a field of the test? This makes the line nosier than needs to be.